modules/control/common/mrac_controller_test.cc

```diff
@@ -70,5 +71,30 @@ TEST_F(MracControllerTest, MracControl) {
   EXPECT_EQ(mrac_controller.Control(-18.0, state, dt), control_value);
 }
 
+// test the judgement of the symetric positive definite solution of the Lyapunov
+// equation
+TEST_F(MracControllerTest, CheckLyapunovPD) {
```

Comment:
Please also add test for first order system check. Since both 1st&2nd order system would potentially be commonly used.

Reply:
fixed per comment. Thanks.